pkg/ipam/metrics/metrics.go

```diff
@@ -14,14 +14,17 @@ import (
 const ipamSubsystem = "ipam"
 
 type prometheusMetrics struct {
-	registry              *prometheus.Registry
-	Allocation            *prometheus.HistogramVec
-	Release               *prometheus.HistogramVec
-	AllocateInterfaceOps  *prometheus.CounterVec
-	AllocateIpOps         *prometheus.CounterVec
-	ReleaseIpOps          *prometheus.CounterVec
-	IPsAllocated          *prometheus.GaugeVec
+	registry             *prometheus.Registry
+	Allocation           *prometheus.HistogramVec
+	Release              *prometheus.HistogramVec
+	AllocateInterfaceOps *prometheus.CounterVec
+	AllocateIpOps        *prometheus.CounterVec
+	ReleaseIpOps         *prometheus.CounterVec
+	IPsAllocated         *prometheus.GaugeVec
+	// Deprecated: Use InterfaceCandidates and EmptyInterfaceSlots instead
```

Comment:
Should we plan to remove it in the next version?

```suggestion
	// Deprecated, will be removed in version 1.14:
	// Use InterfaceCandidates and EmptyInterfaceSlots instead
```